File: web3/rest-with-lombok/src/main/java/luzia/yasmin/ExampleHelloWorld2/model/Product.java
package luzia.yasmin.ExampleHelloWorld2.model;

import lombok.*;

/**
 * @author Yasmin L
 * @since 17/04/2020 - 16:53
 * @version 1.0
 */
@Getter @Setter
@AllArgsConstructor @NoArgsConstructor
@ToString
public class Product {

    private String description;
    private double price;

}

File: mobile1/ExProducts/app/src/main/java/yasmin/luzia/exproducts/model/Product.kt
package yasmin.luzia.exproducts.model

import java.io.Serializable

class Product(var id: Int,
            var name: String,
            var type: String,
            var price: Double) : Serializable {

    override fun toString(): String {
        return "Product(id=$id, name='$name', type='$type', price=$price)"
    }
}